doc/options.rst

```diff
@@ -0,0 +1,115 @@
+Options and file formats
+------------------------
+In the optimization-process of *SMAC*, there are several ways to configure the
+options:
+
+Mandatory:
+    * `Commandline <basic_usage.html#commandline>`_-options, with which *SMAC* is called directly (not needed if
+      *SMAC* is used within Python).
+    * Scenario-options, that are specified via a Scenario-object. Either directly
+      in the Python-code or by using a scenario_-file.
+    * A Parameter Configuration Space (`PCS <options.html#paramcs>`_), that provides the legal ranges of
+      parameters to optimize, their types (e.g. int or float) and their default
+      values.
+
+Optional:
+    * Instance_- and feature_-files, that list the instances and features to
+      optimize upon.
+
+.. _scenario:
+
+Scenario
+~~~~~~~~
+The scenario-object is used to configure *SMAC* and can be constructed either by providing an actual
+scenario-object (see `SVM-example <quickstart.html#using-smac-in-python-svm>`_), or by specifing the options in a
+scenario file (see `SPEAR example <quickstart.html#spear-example>`_).
+The format of the scenario file is one option per line:
+
+.. code-block:: bash
+
+        OPTION1 = VALUE1
+        OPTION2 = VALUE2
+        ...
+
+For boolean options "1" or "true" both evaluate to True.
+The following assumes that the scenario is created via a scenario-file. If it is
+generated within custom code, you might not need *algo* or *paramfile*.
+
+Required:
+        * *run_obj* in [runtime, quality]. Defines what metric to optimize. When optimizing runtime, *cutoff_time* is required as well.
+        * *cutoff_time* is the maximum runtime, after which the target-algorithm is cancelled. **Required if *run_obj* is runtime.**
+        * *algo* specifies the target-algorithm call that *SMAC* will optimize. Interpreted as a bash-command.
+        * *paramfile* specifies the path to the PCS-file
+
+Optional:
+        * *abort_on_first_run_crash* in [true, false]. If true, *SMAC* will abort if the first run of the target algorithm crashes. Default: true.
+        * *execdir* specifies the path to the execution-directory. Default: ".".
+        * *deterministic* in [true, false]. If true, the optimization process will be repeatable. Default: false 
+        * *overall_obj* is PARX, where X is an integer defining the penalty imposed on timeouts (i.e. runtimes that exceed the *cutoff-time*). Default: PAR10.
+        * *memory_limit* is the maximum available memory the target-algorithm can occupy before being cancelled.
+        * *tuner-timeout* is the maximum amount of CPU-time used for optimization. Default: inf.
+        * *wallclock_limit* is the maximum amount of wallclock-time used for optimization. Default: inf.
+        * *runcount_limit* is the maximum number of algorithm-calls during optimization. Default: inf.
+        * *minR* is the minimum number of calls per configuration. Default: 1
```

Comment:
Could you please reword this description? `minimum number of calls` and `maximum number of calls` is not very descriptive if a user doesn't know about the intensify procedure.